### PR TITLE
Fix minor type error in sp_role_mappings_provider_element.adoc

### PR DESCRIPTION
**Pull request by shane-1**
change oro to or

### CODE REVIEW

**Review by andymunro — APPROVED** [review]

**Comment by shane-1** [comment]
change oro to or

**Review by andymunro — APPROVED** [review]

**Review by zlweicoder — APPROVED** [review]
approved

**Review by ahus1 — APPROVED** [review]

**Comment by ahus1** [comment]
@shane-1 - there is no need to rebase this at the moment. Please wait for a maintainer to have this merged. Thanks!

**Pull request merged**
